tests/src/model/pages/dashboardPage.ts

```diff
@@ -0,0 +1,29 @@
+/**********************************************************************
```

Comment:
FYI we usually use convention lowercase-dash like `dashboard-page.ts` for typescript class (that are not in the renderer package)